packages/client-sdk-nodejs/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "target": "ES2018",
+    "target": "ES2022",
     "module": "commonjs",
-    "lib": ["es2018", "dom"],
+    "lib": ["es2022", "dom"],
```

Comment:
this is probably fine but just curious what prompted the upgrade

Reply:
I can't use the BigInt() constructor for targets below ES2020 apparently. I figured I'd update it to a more recent target

Reply:
we have to be careful about whether this has implications for our users, though; does this mean that people can't use us in their own projects if they are not also targeting this platform?

@bruuuuuuuce can you weigh in here?

Reply:
I don't think changing this lib is a problem, but I think changing the `target` might be an issue. That is saying we are targeting to compile down to es2022, which I believe means it might not work in nodejs 16. We might need to add a bigint pollyfill for older node versions which don't support

Reply:
https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping

looks like es2022 targets node version 20

Reply:
Okay. Also I was mistaken, I can still use `BigInt()` constructor with target set to ES2018, I just can't use BigInt literals like `123n`

Reply:
That makes sense, I looked up bigint and it was released in version 10.4, which is target `es2018`. So I think we should be good with keeping it as is

Reply:
Okay I'll switch them both back to `es2018` and just switch to using the constructor

Reply:
💯